src/Tizen.NUI/src/public/Accessibility/Accessibility.cs

```diff
@@ -121,6 +121,19 @@ public void StopReading(bool alsoNonDiscardable)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Suppress reading of screen-reader
+        /// </summary>
+        /// <param name="suppress">whether to suppress reading of screen-reader</param>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool SuppressScreenReader(bool suppress)
+        {
+            bool ret = Interop.Accessibility.SuppressScreenReader(View.getCPtr(dummy), suppress);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
```

Comment:
could you check the warning shown in line 197 ?
you can pull out this EventArgs class from this class. and then you can make new file or you can add this in this file. 

also, could you check the line 222 below?
it is hidden api so, it can be changed as `SayFinishedState`.
thank you.